Index: backend/build/resources/main/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>jwt.expiration-millis=86400000\r\njwt.issuer=securityApp\r\njwt.secret=wIDAQABAkAzGlFARFau7K0b9FcEwIDAQABAkAzGlFARFau7K0b9FcEwIDAQABAkAzGlFARFau7K0b9FcE\r\nspring.datasource.driver-class-name=org.postgresql.Driver\r\nspring.datasource.type=com.zaxxer.hikari.HikariDataSource\r\nspring.datasource.url=jdbc:postgresql://localhost:5432/postgres\r\nspring.datasource.username=postgres\r\nspring.datasource.password=postgrespw\r\nspring.jpa.hibernate.ddl-auto=create
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/backend/build/resources/main/application.properties b/backend/build/resources/main/application.properties
--- a/backend/build/resources/main/application.properties	(revision 94f29eab2cc8c0e3c989fafbcd26655049bbd3aa)
+++ b/backend/build/resources/main/application.properties	(date 1702119807755)
@@ -5,5 +5,5 @@
 spring.datasource.type=com.zaxxer.hikari.HikariDataSource
 spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
 spring.datasource.username=postgres
-spring.datasource.password=postgrespw
-spring.jpa.hibernate.ddl-auto=create
\ No newline at end of file
+spring.datasource.password=postgres
+spring.jpa.hibernate.ddl-auto=create
